src/tennisSimulator/Serve.java

```diff
@@ -0,0 +1,46 @@
+package tennisSimulator;
+
+public class Serve {
```

Comment:
Overall this class looks well designed. It does only what it is supposed to and does not know too much information about other classes.

Well done!